campair/campair/UseCase/EditorUseCase.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  UseCasePlaceHolder.swift
+//  campair
+//
+//  Created by Shin Jae Ung on 2022/06/10.
+//
+
+import Foundation
+import OSLog
+
+struct EditorUseCase {
+    let repository: EditorcontentsFetchable
```

Comment:
```suggestion
    private let repository: EditorcontentsFetchable
```

struct 내에서만 필요한 정수는 접근제한자 설정이 좋을 것 같아용

Reply:
@unnnyong 감사합니다! 중요한 부분이죠.

Reply:
Thanks! yong!